### PR TITLE
Added 47 Libraries to directory 🚀

### PR DESCRIPTION
**Pull request by mitulsavani**
### UI

* [react-native-material-dropdown](https://github.com/n4kz/react-native-material-dropdown)
* [react-native-open-maps](https://github.com/brh55/react-native-open-maps)
* [recyclerlistview](https://github.com/Flipkart/recyclerlistview)
* [react-native-spinkit](https://github.com/maxs15/react-native-spinkit)
* [react-native-mapbox-gl](https://github.com/nitaliano/react-native-mapbox-gl)
* [react-virgin](https://github.com/Trixieapp/react-virgin)
* [react-native-orientation](https://github.com/yamill/react-native-orientation)
* [react-native-tableview](https://github.com/aksonov/react-native-tableview)
* [react-native-largelist](https://github.com/bolan9999/react-native-largelist)
* [react-native-offline](https://github.com/rgommezz/react-native-offline)
* [react-native-root-toast](https://github.com/magicismight/react-native-root-toast)
* [react-native-masonry](https://github.com/brh55/react-native-masonry)
* [react-native-sglistview](https://github.com/sghiassy/react-native-sglistview)
* [react-native-timeline-listview](https://github.com/thegamenicorus/react-native-timeline-listview)
* [react-native-copilot](https://github.com/mohebifar/react-native-copilot)
* [react-native-star-rating](https://github.com/djchie/react-native-star-rating)
* [react-native-sketch](https://github.com/jgrancher/react-native-sketch)
* [galio](https://github.com/galio-org/galio)
* [react-native-super-grid](https://github.com/saleel/react-native-super-grid)
* [react-native-activity-view](https://github.com/naoufal/react-native-activity-view)
* [react-native-refresher](https://github.com/syrusakbary/react-native-refresher)
* [react-native-carousel](https://github.com/nick/react-native-carousel)

### Text & Rich

* [react-native-draftjs-render](https://github.com/globocom/react-native-draftjs-render)
* [react-native-html-to-pdf](https://github.com/christopherdro/react-native-html-to-pdf)
* [react-native-html-webview](https://github.com/almost/react-native-html-webview)
* [react-native-html-render](https://github.com/soliury/react-native-html-render)
* [react-native-measure-text](https://github.com/airamrguez/react-native-measure-text)
* [react-native-file-viewer](https://github.com/vinzscam/react-native-file-viewer)
* [react-native-responsive-fontsize](https://github.com/heyman333/react-native-responsive-fontsize)
* [react-native-text-size](https://github.com/aMarCruz/react-native-text-size)
* [react-native-hypertext](https://github.com/agentcooper/react-native-hypertext)
* [react-native-asciimage](https://github.com/turley/react-native-asciimage)
* [react-native-parsed-text](https://github.com/taskrabbit/react-native-parsed-text)

### Analytics

* [react-native-fabric](https://github.com/corymsmith/react-native-fabric)
* [react-native-ux-cam](https://github.com/uxcam/react-native-ux-cam)
* [react-native-td](https://github.com/quipper/react-native-td)
* [react-native-flurry-sdk](https://github.com/flurry/react-native-flurry-sdk)

### Other Platforms
* [react-native-web](https://github.com/necolas/react-native-web)
* [react-native-macos](https://github.com/ptmt/react-native-macos)
* [reactxp](https://github.com/Microsoft/reactxp)
* [react-native-windows](https://github.com/microsoft/react-native-windows)

### Deep Link

* [react-native-deep-link](https://github.com/Starotitorov/react-native-deep-link)

### Geolocation
* [react-native-geo-fencing](https://github.com/surialabs/react-native-geo-fencing)
* [react-native-fused-location](https://github.com/MustansirZia/react-native-fused-location)
* [react-native-google-static-map](https://github.com/yelled3/react-native-google-static-map)
* [react-native-boundary](https://github.com/eddieowens/react-native-boundary)
* [react-native-geolocation-android](https://github.com/lxsameer/react-native-geolocation-android)

 
# Checklist

If you added a new library:

- [ X] Added it to **react-native-libraries.json**

### CODE REVIEW

**Comment by mitulsavani** [comment]
Hi @brentvatne Please let me know if everything looks good to you

File: react-native-libraries.json [hunk 1]
```diff
@@ -3959,5 +3959,337 @@
...
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-open-maps",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/Flipkart/recyclerlistview",
+    "ios": true,
+    "android": true,
+    "web": true,
```
Comment (brentvatne) [inline react-native-libraries.json:3981]:
```suggestion
    "web": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 2]
```diff
@@ -3959,5 +3959,337 @@
...
+    "githubUrl": "https://github.com/n4kz/react-native-material-dropdown",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-material-dropdown",
+    "unmaintained": true
+  },
+  {
+    "githubUrl": "https://github.com/brh55/react-native-open-maps",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:3973]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 3]
```diff
@@ -3959,5 +3959,337 @@
     "windows": false,
     "npmPkg": "react-native-purchases",
     "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/n4kz/react-native-material-dropdown",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:3966]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 4]
```diff
@@ -3959,5 +3959,337 @@
...
+    "githubUrl": "https://github.com/nitaliano/react-native-mapbox-gl",
+    "ios": true,
+    "android": true,
+    "npmPkg": "@mapbox/react-native-mapbox-gl",
+    "unmaintained": true
+  },
+  {
+    "githubUrl": "https://github.com/Trixieapp/react-virgin",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4007]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 5]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "",
```
Comment (brentvatne) [inline react-native-libraries.json:4008]:
```suggestion
```

File: react-native-libraries.json [hunk 6]
```diff
@@ -3959,5 +3959,337 @@
...
+    "githubUrl": "https://github.com/yamill/react-native-orientation",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-orientation",
+    "unmaintained": true
+  },
+  {
+    "githubUrl": "https://github.com/aksonov/react-native-tableview",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4021]:
```suggestion
    "android": false,
```

File: react-native-libraries.json [hunk 7]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-tableview",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/bolan9999/react-native-largelist",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4028]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 8]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-largelist",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/rgommezz/react-native-offline",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4035]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 9]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-offline",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/magicismight/react-native-root-toast",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4042]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 10]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-root-toast",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/brh55/react-native-masonry",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4049]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 11]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-masonry",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/sghiassy/react-native-sglistview",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-sglistview",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4058]:
```suggestion
    "expo": true,
    "unmaintained": true
```

File: react-native-libraries.json [hunk 12]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/thegamenicorus/react-native-timeline-listview",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-timeline-listview",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4065]:
```suggestion
    "expo": true,
    "unmaintained": true
```

File: react-native-libraries.json [hunk 13]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/mohebifar/react-native-copilot",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4070]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 14]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-copilot",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/djchie/react-native-star-rating",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-star-rating",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4079]:
```suggestion
    "expo": true,
    "unmaintained": true
```

File: react-native-libraries.json [hunk 15]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/jgrancher/react-native-sketch",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-sketch",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4086]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 16]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/galio-org/galio",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4091]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 17]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "galio-framework",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/saleel/react-native-super-grid",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4098]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 18]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-super-grid",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/naoufal/react-native-activity-view",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-activity-view",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4107]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 19]
```diff
@@ -3959,5 +3959,337 @@
...
+    "android": true,
+    "npmPkg": "react-native-refresher",
+    "unmaintained": true
+  },
+  {
+    "githubUrl": "https://github.com/nick/react-native-carousel",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-carousel",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4121]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 20]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/globocom/react-native-draftjs-render",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-draftjs-render",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4128]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 21]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/christopherdro/react-native-html-to-pdf",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-html-to-pdf",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4135]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 22]
```diff
@@ -3959,5 +3959,337 @@
...
+    "android": true,
+    "npmPkg": "react-native-html-render",
+    "unmaintained": true
+  },
+  {
+    "githubUrl": "https://github.com/airamrguez/react-native-measure-text",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-measure-text",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4157]:
```suggestion
    "unmaintained": true
```

Comment (brentvatne) [reply react-native-libraries.json:4157]: https://github.com/airamrguez/react-native-measure-text/issues/18

File: react-native-libraries.json [hunk 23]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/vinzscam/react-native-file-viewer",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4162]:
```suggestion
    "android": true,
    "windows": true,
```

File: react-native-libraries.json [hunk 24]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-file-viewer",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/heyman333/react-native-responsive-fontsize",
+    "ios": true,
+    "android": true,
```
Comment (brentvatne) [inline react-native-libraries.json:4169]:
```suggestion
    "android": true,
    "expo": true,
```

File: react-native-libraries.json [hunk 25]
```diff
@@ -3959,5 +3959,337 @@
...
+    "npmPkg": "react-native-responsive-fontsize",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/aMarCruz/react-native-text-size",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-text-size",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4178]:
```suggestion
    "unmaintained": true
```

Comment (brentvatne) [reply react-native-libraries.json:4178]: https://github.com/aMarCruz/react-native-text-size/commit/b7c722a2b0e3fdd421acc1b8159b0ce8fc6acdcc is the most recent commit, a stable release that supports 0.60 was never published

File: react-native-libraries.json [hunk 26]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/agentcooper/react-native-hypertext",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-hypertext",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4185]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 27]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/turley/react-native-asciimage",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-asciimage",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4192]:
```suggestion
    "unmaintained": true
```

Comment (brentvatne) [reply react-native-libraries.json:4192]: this hasn't been updated in 5 years.. please be more careful when submitting libraries to try to include correct metadata

File: react-native-libraries.json [hunk 28]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/taskrabbit/react-native-parsed-text",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-parsed-text",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4199]:
```suggestion
    "unmaintained": true
```

https://github.com/taskrabbit/react-native-parsed-text/issues/77

File: react-native-libraries.json [hunk 29]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/corymsmith/react-native-fabric",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-fabric",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4206]:
```suggestion
    "unmaintained": true
```

fabric no longer exists... https://github.com/corymsmith/react-native-fabric/issues/232

File: react-native-libraries.json [hunk 30]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/uxcam/react-native-ux-cam",
+    "ios": true,
+    "android": true,
+    "npmPkg": "",
```
Comment (brentvatne) [inline react-native-libraries.json:4212]:
```suggestion
```

please don't include an empty npm package name

File: react-native-libraries.json [hunk 31]
```diff
@@ -3959,5 +3959,337 @@
...
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/quipper/react-native-td",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-td",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4220]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 32]
```diff
@@ -3959,5 +3959,337 @@
...
+    "android": true,
+    "npmPkg": "react-native-flurry-sdk",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/Starotitorov/react-native-deep-link",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-deep-link",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4234]:
```suggestion
    "unmaintained": true
```

not updated in over a year

File: react-native-libraries.json [hunk 33]
```diff
@@ -3959,5 +3959,337 @@
...
+    "windows": true,
+    "npmPkg": "react-native-windows",
+    "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/surialabs/react-native-geo-fencing",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-geo-fencing",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4267]:
```suggestion
    "unmaintained": true
```

not updated since 2016

File: react-native-libraries.json [hunk 34]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/MustansirZia/react-native-fused-location",
+    "android": true,
+    "npmPkg": "react-native-fused-location",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4273]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 35]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/yelled3/react-native-google-static-map",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-google-static-map",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4280]:
```suggestion
    "unmaintained": true
```

File: react-native-libraries.json [hunk 36]
```diff
@@ -3959,5 +3959,337 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/eddieowens/react-native-boundary",
+    "ios": true,
+    "android": true,
+    "npmPkg": "react-native-boundary",
+    "unmaintained": false
```
Comment (brentvatne) [inline react-native-libraries.json:4287]:
```suggestion
    "unmaintained": true
```

**Comment by mitulsavani** [comment]
I appreciate you for your time and efforts in reviewing the PR. I will work on your feedback soon! 🙏 

Initially, I was under the impression that I would have to change the `unmaintained` tag as `true` if the repo is officially marked as deprecated. I will definitely watch out for that.

Thanks once again

File: react-native-libraries.json (hunk 27 above)
Comment (mitulsavani) [reply react-native-libraries.json:4192]: my bad, will be a bit careful next time

**Comment by brentvatne** [comment]
hey @mitulsavani! it can be a bit tricky to determine if it's compatible with expo client, i tried to help explain that here: https://docs.expo.io/workflow/using-libraries/#determining-third-party-library-compatibility. if you're not sure, you can leave it blank and i will fill it in the pull requests, but an effort to determine whether it is or not is definitely much appreciated!

**Pull request merged**